discovery-provider/alembic/trigger_sql/handle_repost.sql

```diff
@@ -105,6 +105,55 @@ begin
       on conflict do nothing;
     end if;
 
+	-- notify followees of the reposter who have reposted the same content
+	-- within the last month
+	if new.is_delete is false
+	and new.is_repost_repost is true then
+	with
+	    followee_repost_repost_ids as (
+	        select user_id
+	        from reposts r
+	        where
+	            r.repost_item_id = new.repost_item_id
+	            and new.created_at + INTERVAL '-1 month' < r.created_at
+	            and new.created_at > r.created_at
+				and r.is_delete is false
```

Comment:
some indentation is off here so reading the query can be confusing

Reply:
weird.. i fixed this on my local branch but idt git is picking up the changes bc its saying everything is up to date with the remote branch?